server/mscalendar/daily_summary.go

```diff
@@ -75,6 +83,14 @@ func (m *mscalendar) SetDailySummaryEnabled(user *User, enable bool) (*store.Dai
 		return nil, err
 	}
 
+	if user.Settings.DailySummary == nil {
+		user.Settings.DailySummary = &store.DailySummaryUserSettings{
```

Comment:
LGTM. Wondering if it would be worth returning this object from a small function since it is used 3 times?

Reply:
I thought so, but since the object is being used in the store and in the main package, I thought it would create circular dependencies unless we store it in the store. And the store does not seem the best place for this.

Reply:
1/5 I agree with Jason to make it into a function. If the store is the only place it can go, then I suppose that's where it has to go.